lib/utils.js

```diff
@@ -219,8 +218,12 @@ var Utils = module.exports = {
   },
 
   toDefaultValue: function(value) {
-    if (lodash.isFunction(value)) {
-      return value();
+    if (typeof value === 'function') {
+      if (value.key) {
+        return value().key;
```

Comment:
what is `value.key`?
If it's to detect a sequelize method use `_isSequelizeMethod` instead and then the proper handler (usually toString())